### PR TITLE
Fix visibility of def_parser for remote builds

### PR DESCRIPTION
**Pull request by meteorcloudy**
Fix https://github.com/bazelbuild/bazel/issues/4883

### CODE REVIEW

**Comment by laszlocsomor** [comment]
Wait, what's up with the BuildKite failure?

**Comment by meteorcloudy** [comment]
Hmm,, I think we cannot send PR for files under third_party

**Comment by meteorcloudy** [comment]
If you LGTM, I can manually push to GoB

**Comment by meteorcloudy** [comment]
FYI, @jasharpe

**Comment by laszlocsomor** [comment]
Can you send a commit on Gerrit then?

**Comment by meteorcloudy** [comment]
Yes, but we still have to push manually and it won't trigger any tests either, so I guess there's no point to do that.

**Comment by laszlocsomor** [comment]
Ok, SGTM.

File: third_party/def_parser/BUILD [hunk 1]
```diff
@@ -1,5 +1,12 @@
 licenses(["notice"])  # 3-clause BSD
 
+package(
+    default_visibility = [
+        "//src:__subpackages__",
+        "//tools/def_parser:__subpackages__",
```
Comment (jasharpe) [inline third_party/def_parser/BUILD]: Will this work even though there's an explicit visibility declaration on the :def_parser rule below?

Comment (meteorcloudy) [reply third_party/def_parser/BUILD]: Oops, probably not, thanks for spotting this.

**Pull request closed without merging**